charts/logzio-monitoring/Chart.yaml

```diff
@@ -12,7 +12,7 @@ dependencies:
     repository: "https://logzio.github.io/logzio-helm/"
     condition: logs.enabled
   - name: logzio-k8s-telemetry
-    version: "0.0.2"
+    version: "0.0.3"
```

Comment:
Since you're bumping the dependency please also bump logzio-monitoring's Chart version, and update its readme's changlog.